src/Core/Domain/Product/Command/UpdateProductPackCommand.php

```diff
@@ -45,17 +45,39 @@ class UpdateProductPackCommand
     /**
      * @var QuantifiedProduct[]
      */
-    private $products;
+    private $products = [];
 
     /**
+     * Builds command to remove all items from the pack
+     *
+     * @param int $packId the id of product which becomes the pack after it contains packed items
+     *
+     * @return UpdateProductPackCommand
+     */
+    public static function cleanPack(int $packId): self
+    {
+        return new self($packId, []);
+    }
+
+    /**
+     * Builds command to upsert pack with new products list.
```

Comment:
upsert? 🤔
Like update/insert? Then I would propose `indate` ! It's easier to pronounce! 😂

Reply:
Ok after a quick google search I just learnt it's actually a thing 😅 
In PostgreSQL for instance